src/handlers/comment/handlers/index.ts

```diff
@@ -36,6 +43,21 @@ export * from "./help";
 export * from "./multiplier";
 export * from "./query";
 
+export interface RewardsResponse {
+  error: string | null;
+  title?: string;
+  user_id?: string;
```

Comment:
```suggestion
  userId?: string;
```

Reply:
Does that matter?

Reply:
Well we should try to be consistent and use camelCase everywhere